Overlay/style.css

```diff
@@ -0,0 +1,12 @@
+@import '../variables.css';
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--outer-space-ultra-dark);
```

Comment:
Curious about `darkenedOverlay`; might this be a new name for our `--outer-space-ultra-dark` variable?